package.json

```diff
@@ -49,7 +49,7 @@
     "build-templates": "node ./src/Common/Connectivity/Dependencies.generator.js",
     "test": "cross-env TEST_SUITE_MOCK_ONLY=1 TEST_SUITE_HEADLESS=1 LOG_LEVEL=silent LOG_SILENT=1 mocha \"dist/Tests/**/*spec.js\"",
     "test-coverage": "cross-env TEST_SUITE_MOCK_ONLY=1 TEST_SUITE_HEADLESS=1 LOG_LEVEL=silent LOG_SILENT=1 nyc --reporter text --reporter html --reporter cobertura npm run test",
-    "copy-proj": "npx rimraf examples/deployment/docker/.work && npx cpx src/** examples/deployment/docker/.work && npx cpx {package,tsconfig}.json examples/deployment/docker/.work/"
+    "copy-proj": "npx rimraf examples/deployment/docker/.work && npx cpx 'src/**' 'examples/deployment/docker/.work/src' && npx cpx '{package,tsconfig}.json' 'examples/deployment/docker/.work'"
```

Comment:
Seemed like the docker image was missing the `src` folder to nest everything properly, but perhaps I've missed something

Reply:
Yep, i had similar change on my local, forgot to push it back. i believe this is an issue with mac user (I'm using mac).
The original command work on Windows terminal but not mac. 
I just test out your change it works on my local as well.

Reply:
Yeah that's a good option, another option I'm wondering is we check if this will run on windows with the quotes, I've not got a windows machine on hand, but I could get a vm setup to try it out.

Reply:
yeh no windows machine either, may have to spin up a VM to test it. Give it a go if you don't mind spending a bit of time :)

Reply:
I'll give it a go on windows. I did notice that this was an issue in the `test` task, so it will probably be fine if we stick with the double quotes, I would guess.

Reply:
By the way, @mathewdbutton, we absolutely encourage and welcome contributions like these from the community - thank you!

Reply:
Hi @mathewdbutton,

I've tested with windows and the single quotes are a no go, unfortunately. Hopefully double quotes are. Can you please check that the following will work for you?

```
"copy-proj": "npx rimraf examples/deployment/docker/.work && npx cpx \"src/**\" \"examples/deployment/docker/.work/src\" && npx cpx \"{package,tsconfig}.json\" \"examples/deployment/docker/.work\""
```

Thanks again.

Reply:
On it @pcurtisrab, single quotes was because of laziness from my end. I'll try it now.

Reply:
Bingo! it's working! lets see if CI likes the change of quotes.